README.md

```diff
@@ -79,20 +80,31 @@ Install this project with:
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ### Usage
+
 > To run the project, execute the following command:
+
 ```sh
    rails server
 ```
 
 ## 👥 Authors <a name="authors"></a>
 
-
 👤 **BAHATI Marie-Grace**
 
 - GitHub: [@mariegrace31](https://github.com/mariegrace31)
 - Twitter: [@mariegracebmg](https://twitter.com/mariegracebmg)
 - LinkedIn: [Marie Grace Bahati](www.linkedin.com/in/marie-grâce-bahati)
 
```

Comment:
![image](https://github.com/mariegrace31/blogApp/assets/93039359/0a99f30b-f987-4a9a-9470-259316819029)

- [ ] It would be nice if you fix the highlighted link in your readme file to make your readme descriptive and professional 👍